README.md

```diff
@@ -59,3 +59,11 @@ set the following in your app environment:
 ```sh
 heroku config:set NPM_CONFIG_PRODUCTION=false
 ```
+
+## Using the latest source code
+
+The `zeke/npm` buildpack from the [Heroku Buildpack Registry](https://devcenter.heroku.com/articles/buildpack-registry) represents the latest stable version of the buildpack. If you'd like to use the source code from this Github repository, you can set your buildpack to the Github URL:
+
+```sh-session
```

Comment:
I've never seen `sh-session` before. How does this differ from `sh`?

Reply:
It's supposed to be shell session (like with a prompt) and it does an ok job at colorizing some keywords. Fun fact, I added it ! :) https://github.com/github/linguist/pull/1071